src/grid.tsx

```diff
@@ -59,6 +59,181 @@ if ((module as any).hot) {
 }
 /* eslint-enable */
 
+/**
+ * Grid public API
+ * @param {Object} options
+ *      @param {Array} [options.data] - Grid data for making rows.
```

Comment:
`@param {Array} [options.data] - Grid data for making rows.` -> `@param {Array| Object} [options.data] - Grid data for making rows.`